2/contrib/s2i/run

```diff
@@ -105,6 +105,28 @@ function migrate_slave_configuration() {
   sed -i "s#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7\(:.*\)\{0,1\}</image>#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7:${JENKINS_SLAVE_IMAGE_TAG}</image>#" /var/lib/jenkins/config.xml
 }
 
+function force_copy_plugins() {
+   # if rhel
+   if [ "$(ls /usr/lib/jenkins/*.hpi 2>/dev/null)" ]; then
+      echo "This openshift jenkins image is starting in a deployment where the image version"
+      echo "was not recoreded.  A one time replacement of all the plugins from the image to the"
```

Comment:
s/recoreded/recorded/

Reply:
also this message is going to be printed when the imageversion is older than the current image also, so the text needs to be reworked.

Reply:
Message reworded, including dumping of the image and deployment versions.

part of next push